### PR TITLE
Updates to GMT, Language moderators, Support team and osu! alumni

### PR DESCRIPTION
**Pull request by WalterToro**
Very long overdue

---

### CODE REVIEW

File: wiki/People/Global_Moderation_Team/en.md [hunk 1]
```diff
@@ -16,38 +16,41 @@ The Global Moderation Team is responsible for the welfare of the chat/forum and
...
 | ![][flag_BR] [Edu](https://osu.ppy.sh/users/5618109) | Portuguese, Japanese | Chat Moderation |
-| ![][flag_KR] [Elfin](https://osu.ppy.sh/users/1399551) | Korean, Japanese | Chat Moderation, Forum Moderation |
 | ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) | | Player Support |
 | ![][flag_BG] [Flanster](https://osu.ppy.sh/users/447818) | Bulgarian | osu!idol hosting, Forum Moderation |
 | ![][flag_CN] [Fycho](https://osu.ppy.sh/users/1876867) | Chinese | Chat Moderation, Forum Moderation |
 | ![][flag_CA] [Gabe](https://osu.ppy.sh/users/654108) | French | Chat Moderation, Forum Moderation |
 | ![][flag_PL] [Galkan](https://osu.ppy.sh/users/169570) | Polish | Chat Moderation |
 | ![][flag_BR] [ghm12](https://osu.ppy.sh/users/2594229) | Portuguese | Chat Moderation |
 | ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779) | Turkish | Chat Moderation |
+| ![][flag_ID] [Hinsvar](https://osu.ppy.sh/users/1249323) | Indonesian | Chat Moderation, BSS Moderation |
```
Comment (cl8n) [inline wiki/People/Global_Moderation_Team/en.md]: probably `Beatmap Moderation` instead

Comment (WalterToro) [reply wiki/People/Global_Moderation_Team/en.md]: This is due to to the fact that we will moderate elements of the beatmaps themselves, just the content of the uploads.

**Comment by WalterToro** [comment]
Also, this is blocked due to the fact i need to update the rest of the language files, and other related articles.

File: wiki/People/Global_Moderation_Team/en.md (hunk 1 above)
Comment (cl8n) [reply wiki/People/Global_Moderation_Team/en.md]: not sure what the difference is but either way you're definitely not moderating BSS itself so... :thinking:

Comment (peppy) [reply wiki/People/Global_Moderation_Team/en.md]: "Beatmap report processing" or something?

Comment (WalterToro) [reply wiki/People/Global_Moderation_Team/en.md]: I will go with that

**Comment by WalterToro** [comment]
Updated most of the files but i need to request a translation for "Beatmap moderation" or "Beatmap management" to chinese so i can finish that file.

**Comment by WalterToro** [comment]
Finally done and updated, ready for merge

**Pull request merged**